pkg/trace/call.go

```diff
@@ -96,6 +97,15 @@ func SetCallError(ctx context.Context, err error) error {
 	return SetCallStatus(ctx, err)
 }
 
+// SetCustomCallKind is meant to alter Kind property of traced call.
+// This can be useful as a service specific dimension to slice e.g. grpc_request_handled metric based on additional context
```

Comment:
Nit: wording
```suggestion
// SetCustomCallKind is meant to alter the Kind property of a traced call.
// This can be useful as a service specific dimension to slice, e.g.
// grpc_request_handled metric based on additional context.
```